services/frontend-v3/src/components/profileForms/employmentDataForm/EmploymentDataFormView.jsx

```diff
@@ -721,11 +721,12 @@ const EmploymentDataFormView = ({
               <DescriptionFormItem
                 name="description"
                 fieldKey="description"
-                rule={Rules.maxChar1000}
-                value={profileInfo.description}
+                rules={[Rules.maxChar1000]}
+                maxLength={1000}
```

Comment:
The only rule i can imagine this field having is the manLength, isRequired, minLength, can we somehow make props for there an avoid passing in "rules". Eventually we are moving the rules to as separate file anyways.

Reply:
The user can now pass the following props instead of rules:
 - maxLength
 - maxLengthMessage
 - minLength
 - minLengthMessage
 - isRequired
 - isRequiredMessage